ssh/manifests/server/config_users.pp

```diff
@@ -0,0 +1,53 @@
+class ssh::server::config_users (
+  $users = $quickstack::params::moc_users
+) {
+
+  define config_user ($user_config = $title) {
```

Comment:
If I understand title correctly, doesn't this set $user_config = config user? How does this make a dictionary out of the three params in the hiera file?